### PR TITLE
test(js): Convert InviteMembersModal to RTL

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

File: static/app/components/modals/inviteMembersModal/inviteRowControl.tsx [hunk 1]
```diff
@@ -79,8 +79,9 @@ function InviteRowControl({
   };
 
   return (
-    <div className={className}>
+    <li className={className}>
```
Comment (priscilawebdev) [inline static/app/components/modals/inviteMembersModal/inviteRowControl.tsx:82]: maybe we can replace this with our list component?

File: static/app/components/modals/inviteMembersModal/index.spec.jsx [hunk 2]
```diff
@@ -40,88 +43,69 @@ describe('InviteMembersModal', function () {
   });
 
   it('renders', function () {
-    const wrapper = mountWithTheme(
-      <InviteMembersModal {...modalProps} organization={org} />
-    );
+    render(<InviteMembersModal {...modalProps} organization={org} />);
```
Comment (priscilawebdev) [inline static/app/components/modals/inviteMembersModal/index.spec.jsx:46]: aha! this is not yet in `ts` and that is why the `modalProps` mock still works here

**Review by priscilawebdev — APPROVED** [review]
great conversion

File: static/app/components/modals/inviteMembersModal/index.spec.jsx (hunk 2 above)
Comment (evanpurkhiser) [reply static/app/components/modals/inviteMembersModal/index.spec.jsx:46]: yeah. What we should do in the future, is export `commonModalProps` or `makeModalProps` from our test utils that can set them up for you

File: static/app/components/modals/inviteMembersModal/inviteRowControl.tsx (hunk 1 above)
Comment (evanpurkhiser) [reply static/app/components/modals/inviteMembersModal/inviteRowControl.tsx:82]: nah, our list component has styling, I just wanted this to be a list

Comment (evanpurkhiser) [reply static/app/components/modals/inviteMembersModal/inviteRowControl.tsx:82]: doesn't it?

Comment (priscilawebdev) [reply static/app/components/modals/inviteMembersModal/inviteRowControl.tsx:82]: yes it has some padding left

**Pull request merged**